example.yaml

```diff
@@ -1,81 +1,163 @@
 ---
-api_hostname: https://app.datadoghq.com
 metric_max_length: 4096
 trace_max_length_bytes: 16384
 flush_max_per_body: 25000
-debug: true
 enable_profiling: false
 interval: "10s"
-key: "farts"
+
+# == PERFORMANCE ==
+
+# The size of the buffer we'll use to buffer socket reads. Tune this if you
+# you think Veneur needs more room to keep up with all packets.
+read_buffer_size_bytes: 2097152
+
+# Adjusts the number of workers Veneur will distribute aggregation across.
+# More decreases contention but has diminishng returns.
+num_workers: 96
+
 # Numbers larger than 1 will enable the use of SO_REUSEPORT, make sure
 # this is supported on your platform!
-num_workers: 96
 num_readers: 1
+
+# The number of SSF packets that can be processed
+# per flush interval
+ssf_buffer_size: 16384
+
+# == AGGREGATION ==
+
+# Set to floating point values that you'd like to putput percentiles for from
+# histograms.
 percentiles:
   - 0.5
   - 0.75
   - 0.99
+
+# Aggregations you'd like to putput for histograms. Possible values can be any
+# or all of:
+# - `min`: the minimum value in the histogram during the flush period
+# - `max`: the maximum value in the histogram during the flush period
+# -	`median`: the median value in the histogram during the flush period
+# -	`avg`: the average value in the histogram during the flush period
+# -	`count`: the number of values added to the histogram during the flush period
+# -	`sum`: the sum of all values added to the histogram during the flush period
 aggregates:
  - "min"
  - "max"
  - "count"
-read_buffer_size_bytes: 2097152
+
+# == DIAGNOSTICS ==
+
+# Enable debug logging
+debug: true
+
+# Providing a Sentry DSN here will send internal exceptions to Sentry
+sentry_dsn: ""
+
+# Where to send metrics about ourselves, can point this at ourself too!
```

Comment:
This is a little unclear - I think "where to send metrics about veneur. (You can point this to the local veneur)" or something along those lines is a little more clear